usaspending_api/common/helpers/endpoint_documentation.py

```diff
@@ -0,0 +1,118 @@
+import os
+import re
+
+from usaspending_api.settings import BASE_DIR
+
+
+# Typically we only care about v2 API endpoints, but if we ever add v3 or
+# whatever, add the base path to this tuple.
+CURRENT_ENDPOINT_PREFIXES = ("/api/v2/",)
+
+ENDPOINTS_MD = "usaspending_api/api_docs/markdown/endpoints.md"
```

Comment:
What about API Contracts? We are moving toward exclusively using them for api documents

Reply:
I don't understand what you are asking.

Reply:
that makes two of us....